lib/block-supports/layout.php

```diff
@@ -145,6 +145,35 @@ function gutenberg_get_layout_style( $selector, $layout, $has_block_gap_support
 	return $style;
 }
 
+/**
+ * Generates the utility classnames for the given blocks layout attributes.
+ *
+ * @param array $block_attributes Array of block attributes.
+ *
+ * @return array Array of CSS classname strings.
+ */
+function gutenberg_get_layout_classes( $block_attributes ) {
+	$class_names = array();
+
+	if ( empty( $block_attributes['layout'] ) ) {
+		return $class_names;
+	}
+
+	if ( ! empty( $block_attributes['layout']['orientation'] ) ) {
+		$class_names[] = 'is-' . $block_attributes['layout']['orientation'];
```

Comment:
Should we wrap each of the `$block_attributes['layout']['orientation']` lines in this function in a call to [sanitize_title](https://developer.wordpress.org/reference/functions/sanitize_title/) to ensure it's slugified? E.g.

```suggestion
		$class_names[] = 'is-' . sanitize_title( $block_attributes['layout']['orientation'] );
```

Reply:
done.